pkgs/data/fonts/gandom-fonts/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "gandom-fonts";
```

`name` is of the form `PackageName-Version`, you might mean `pname` here
```suggestion
  pname = "gandom-fonts";
```